assets/js/components.js

```diff
@@ -47,6 +47,27 @@ jQuery(function () {
         $(td).html(`${rowData.attributes.github_enforce_admins_enabled}`)
       },
     },
+    {
+      data: 'attributes.environments',
+      createdCell: function (td, _cellData, rowData) {
+        const environments = rowData.attributes.environments
+        const prodEnvironments = environments ? environments.filter(env => env.name === 'prod') : []
+
+        let prodSlackChannel = ''
+        if (prodEnvironments.length === 0) {
+          prodSlackChannel = `No Prod environment`
+        } else {
+          prodSlackChannel = prodEnvironments.map(env => {
+            if (env.alerts_slack_channel === null) {
+              return `Missing slack channel data`
+            } else {
```

Comment:
Don't need `else` as you won't get anywhere else in the loop because if the return inside the `if`

So basically:

```
if (env.alerts_slack_channel === null) {
  return `Missing slack channel data`
}

return `<a class="govuk-link--no-visited-state" href="slack://channel?team=T02DYEB3A&id={{ env.alerts_slack_channel }}">${env.alerts_slack_channel}</a>`
```

Reply:
Removed hyperlink